packages/video_player/video_player/pigeons/messages.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:pigeon/pigeon_lib.dart';
```

Comment:
nit, the convention is to have the main dart file be the same name as the package, e.g. package:pigeon/pigeon.dart

Reply:
https://github.com/flutter/flutter/issues/56495